### DISCUSSION TITLE
Main menu doesn't show "Your Enterprises"

### DISCUSSION THREAD

**Discussion by fhmtz-work** [community/community#37564] [Enterprise]
Hi,

We just upgraded to Github Enterprise Cloud but none of our admins are seeing the "Your Enterprises" option in the menu. 

<img width="952" alt="Screen Shot 2022-10-27 at 6 00 00 PM" src="https://user-images.githubusercontent.com/81267109/198399113-752debed-2279-4c12-b5ca-8c210df453ac.png">

<img width="174" alt="Screen Shot 2022-10-27 at 6 12 55 PM" src="https://user-images.githubusercontent.com/81267109/198399195-bfb26abb-fb88-4f45-8f25-e9b15819f5c0.png">

Any ideas about why this is happening and how to solve it?

We need to activate Github Avanced Security code scanning features and without this we are not being able to do that.

Thank you!
Hugo

**Comment by ossanna16** [comment]
Hi there @fhmtz-work and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)

**Comment by islootski** [comment]
Hi
Our organization (vrnspolylab) has the same issue.

**Comment by ozonni** [comment]
same here. Actually we are still on trial and i knew that it will be the same. +1
